Makefile

```diff
@@ -14,6 +14,3 @@ logs:
 
 build:
 	docker build --platform linux/amd64 -t $(name) -f Dockerfile .
-
-delve:
```

Comment:
you found it useless from makefile ?

Reply:
I am an absolute beginner regarding delve. I browse a little bit its doc and I didn't find how to use the command line debugger properly through docker. It may be due to my lack of knowledge on the subject so if you know how to use it in this context, I'd be happy to see it in action 😊

Reply:
no it was just curiosity :) ok for me !